### PR TITLE
RHDEVDOCS-5306: Documenting Pipelines 1.10.2 RN

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve the following issue: 
* https://issues.redhat.com/browse/RHDEVDOCS-5306

**Aligned team**: DevTools

**OCP version this PR applies to (cherrypicking)**: enterprise 4.11 and later

**Content for preview**:  https://59781--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#release-notes-1-10-2_op-release-notes

**SME review** :  @jkandasa 

**QE review**:  @VeereshAradhya 

**Peer review**: @Dhruv-Soni11

### CODE REVIEW

File: modules/op-release-notes-1-10.adoc [hunk 1]
```diff
@@ -166,3 +166,13 @@ With this update, {pipelines-title} General Availability (GA) 1.10.1 is availabl
...
+
+[id="release-notes-1-10-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.2
+
+With this update, {pipelines-title} General Availability (GA) 1.10.2 is available on {product-title} 4.11 and later versions.
+
+[id="fixed-issues-1-10-2_{context}"]
+=== Fixed issues
+
+Before this update, Tekton Operator would not support the `enable-api-fields` flag in the `TektonConfig` CR, which prevented the deployment of the service. With this update, Tekton Operator supports these values, `alpha`, `beta`, and `stable` for the `enable-api-fields` flag in the `TektonConfig` CR.
```
Comment (VeereshAradhya) [inline modules/op-release-notes-1-10.adoc:178]: `enable-api-fields` was supported in previous versions as well. The issue was we could set it only to `alpha` or `stable`. With this fix we can add `beta` value as well.

Comment (jkandasa) [reply modules/op-release-notes-1-10.adoc:178]: We support all these(`alpha`, `beta`, and `stable`) values in previous version as well. Due to a bug in product, user can not set `beta`. We have fixed that issue in this release.

**Review by jkandasa — APPROVED** [review]
LGTM

**Review by VeereshAradhya — APPROVED** [review]

File: modules/op-release-notes-1-10.adoc [hunk 2]
```diff
@@ -166,3 +166,13 @@ With this update, {pipelines-title} General Availability (GA) 1.10.1 is availabl
...
+
+[id="release-notes-1-10-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.2
+
+With this update, {pipelines-title} General Availability (GA) 1.10.2 is available on {product-title} 4.11 and later versions.
+
+[id="fixed-issues-1-10-2_{context}"]
+=== Fixed issues
+
+Before this update, Tekton Operator had a bug that prevented the user from setting the value of the `enable-api-fields` flag to `beta`. This update fixes the issue and you can use the value `beta` for configuring the `enable-api-fields` flag in the `TektonConfig` CR.
```
Comment (Dhruv-Soni11) [inline modules/op-release-notes-1-10.adoc:178]: Can we change this to as follows:

Before this update, an issue in the Tekton Operator prevented the user from setting the value of the `enable-api-fields` flag to `beta`. This update fixes the issue. Now you can set the value to `beta` and configure the `enable-api-fields` flag in the `TektonConfig` CR.

Comment (shipsing) [reply modules/op-release-notes-1-10.adoc]: Incorporated

**Comment by Dhruv-Soni11** [comment]
Hi @Shipsingh, thanks for incorporating the changes. I have checked them and they look fine. LGTM! Good work.

**Pull request merged**